railties/lib/rails/generators/rails/plugin/templates/Gemfile.tt

```diff
@@ -13,7 +13,7 @@ gemspec
 # Start debugger with binding.b [https://github.com/ruby/debug]
 # gem "debug", ">= 1.0.0"
 <% end -%>
-<% if RUBY_PLATFORM.match?(/bccwin|cygwin|emx|mingw|mswin|wince|java/) -%>
+<% if RUBY_PLATFORM.match?(/mingw|mswin|java/) -%>
 
-gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
+gem "tzinfo-data", platforms: %i[ windows jruby ]
```

Comment:
if you prefer, I could template this using ERB to choose between `windows` and `mingw mswin x64_mingw` based on the bundler version support.

Reply:
Yeah, I think there's two ways to go for this: we could either require a minimum version of bundler or make it conditional based on the version.

I'm currently leaning towards wanting a higher minimum bundler version so that people don't end up having to make the change when they eventually upgrade. Since this is just for generating new applications, I don't see much downside to a higher requirement.

Reply:
Done, please check.